drivers/hmis/app/graphql/mutations/base_mutation.rb

```diff
@@ -96,5 +96,13 @@ def default_delete_record(record:, field_name:, permissions: nil)
         errors: [],
       }
     end
+
+    def allowed?(record: nil, permissions: nil, authorize: nil)
+      allowed = true
```

Comment:
Should this default to false?  Am I reading this correctly that if there is no record and no permission provided and we don't send authorize, then we allow it.  Maybe that's correct.

Reply:
Let's document why it works this way.

Reply:
Documented

Reply:
Thanks!